modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -37,17 +37,20 @@ spec:
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <1>
+        - name: bridge-net <2>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName: <network-namespace>/<a-bridge-network> <3>
 ...
 ----
-<1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
-<2> The name of the network attachment definition.
+<1> The name of the bridge interface.
+<2> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
+<3> The name of the Network Attachment Definition, prefixed by the namespace where it exists. The only 2 possible options for this namespace are either the default namespace or the same namespace where the VM is to be created.
```

Comment:
s/Network Attachment Definition/network attachment definition

s/The only 2 possible options for this namespace are/The namespace must be